daliuge-engine/dlg/deploy/configs/__init__.py

```diff
@@ -159,15 +190,15 @@ class Setonix411Config(DefaultConfig):
     ACCOUNT = "pawsey0411"
     USER = os.environ["USER"] if "USER" in os.environ else ""
     HOME_DIR = f"/scratch/{ACCOUNT}"
-    DLG_ROOT = f"{HOME_DIR}/{USER}/dlg"
+    DLG_ROOT = f"{HOME_DIR}/$USER/dlg"
```

Comment:
@awicenec I have added this modification here, as it was causing issues when logging in with different username on Setonix.